plugins/dql-backend/src/service/queryExecutor.ts

```diff
@@ -30,9 +30,13 @@ const componentQueryVariablesSchema = z.object({
     .string()
     .max(63)
     .regex(/^[A-Za-z0-9\-_\.]+$/),
+  additionalFilter: z
+    .string()
+    .optional()
```

Comment:
I think it might be easier to just declare an empty string as a [default](https://zod.dev/?id=default) value as opposed to chaining `optional()` and `transform(..)`.

Reply:
Didn't know that. thx